heareval/score.py

```diff
@@ -142,24 +170,25 @@ class SoundEventScore(ScoreFunction):
     def __init__(
         self,
         label_to_idx: Dict[str, int],
-        score: str,
+        scores: Tuple[str],
         params: Dict = None,
         name: Optional[str] = None,
         maximize: bool = True,
     ):
         """
-        :param score: Score to use, from the list of overall SED eval scores.
+        :param scores: Scores to use, from the list of overall SED eval scores.
+            The first score in the tuple will be the primary score for this metric
         :param params: Parameters to pass to the scoring function,
                        see inheriting children for details.
         """
         if params is None:
             params = {}
         super().__init__(label_to_idx=label_to_idx, name=name, maximize=maximize)
-        self.score = score
+        self.scores = scores
         self.params = params
         assert self.score_class is not None
 
-    def __call__(self, predictions: Dict, targets: Dict, **kwargs):
+    def compute(self, predictions: Dict, targets: Dict, **kwargs):
```

Comment:
Should we add `-> float` to the return type here?

Reply:
return type is not float but a Tuple[Tuple[str, float]]. added